_posts/1993-12-14-story-of-scooty-16.md

```diff
@@ -0,0 +1,118 @@
+the door. “If she goes, they all go!”
+
+Scooty swung the axe once more, and the corpse in front of her doubled over. Blockingjay took his
+head off with one swipe of his sword as Scooty rehung the axe from her pack and started to summon a
+dirt tornado. Jekserah stepped into the center of the mass of corpses and skeletons, summoning two
+more corpses on the way.
+
+“Take this, scum!”
+
+Scooty turned to look at Master Ratchet in surprise and was just in time to see him loose a toxic
+bolt in Jekserah’s direction. The Valrath hissed as it sank into her shoulder, releasing earth
+essence into the air around it.
+
+“A truer shot has never been seen, my dear Quatryl friend!” Something on the other side of Sir
+Integral grunted as his invisible fist connected with its face. “Take THAT, you slimy, no-good,
+son of a-“
+
+Scooty reached out for the earth in the air and began working it into the tornado. Disgust and
+contempt burned in Blockingjay’s eyes as he swept the undead in front of him out of the way, coming
+up beside Jekserah with one long stride. Noticing a hint of fear on the Valrath’s face as
+Blockingjay raised his blade, Scooty let the dirt tornado loose. Swirling chunks of dirt and stone
+hit Jekserah and the undead around her. Blockingjay brought his sword down harder than Scooty had
+ever seen, slicing through Jekserah’s armor. She struggled to raise her bow, staggered by the force
+of Blockingjay’s blow, and shot weakly at the Inox. The dirt tornado circled around again, and a
+chunk of debris hit her in the head.
+
+“What’s taking so long?!” Scooty assumed from the high-pitched shriek that Sir Integral’s
+invisibility cloak had worn out. Two small orbs of fire darted between her and Blockingjay, hitting
+Jekserah with twin sizzles.
+
+Scooty noticed a mechanical grinding as she started the dirt tornado on another rotation.
+Blockingjay brought another mighty blow down on Jekserah’s head, and Sir Integral continued to toss
+various spells into the fray, muttering furiously and turning around every so often to kick or
+swing at an enemy in the next room.
+
+The mechanical grinding was punctuated with a hiss, followed by a thud. Jekserah doubled over,
+holding her stomach, and Scooty was barely able to see another bolt sticking out of her middle.
+Master Ratchet leaped into the air and spun in a celebratory circle, holding a crank bow with both
+hands. Jekserah fell to one knee, overwhelmed by the dirt tornado, and the undead around her froze
+and began to quiver.
+
+“You have no idea what’s coming…” she whispered, life fleeing her eyes.
+
+With one more hack of his blade, Blockingjay separated Jekserah’s head from her body. The remaining
+undead collapsed, suddenly lacking a life force to keep them animated. Scooty turned away from the
+sight of the Valrath’s corpse as Blockingjay picked up the severed head, storing it grimly in an
+unused pocket of his pack.
```

Comment:
Maybe a pouch attached to his pack?  A pocket big enough for a head is a big pocket.  But he is an Inox and huge, so probably has a big pack.